src/locales/enUs/translations/scheduling/index.ts

```diff
@@ -18,18 +16,15 @@ export default {
         emergency: 'Emergency',
         followUp: 'Follow Up',
         routine: 'Routine',
-        walkIn: 'Walk In',
+        walkUp: 'Walk Up',
```

Comment:
Why you changed walk in to walk up? We are using walk in to describe walk-in appointments. Can you please revert this?

Reply:
I am sorry, I actually didn't, it came with the pull I made to master before pushing my local branch. I'll revert that change right away.

Reply:
No worries! You don't have to say sorry! :)